google-cloud-dlp/docs/toc.json

```diff
@@ -1,6 +1,28 @@
 {
   "guides": [],
   "services": [
+    {
```

Comment:
I believe we want to define one service, but have both `V2` and `V2beta1` listed under `nav`.

Reply:
@quartzmo ^ Can you confirm?

Reply:
Sorry, just saw this, taking a look now.

Reply:
@jbolinger Yes, @blowmage is correct. Can you change this to the structure used in [google-cloud-video_intelligence/docs/toc.json](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-video_intelligence/docs/toc.json)?

Reply:
No problem, moved to nav.